### PR TITLE
[monitoring-ping] Skip unschedulable nodes

### PR DESCRIPTION
**Pull request by gmaiorov**
Signed-off-by: Gleb Maiorov <gleb.maiorov@flant.com>

## Description

Skip unschedulabe nodes in `monitoring-ping`.

Changes were tested in local cluster.
After cordoning one of the workers corresponding metrics won't be captured anymore: 
metric-example

## Why do we need it, and what problem does it solve?

Currently unschedulable nodes are still pinged in monitoring-ping module.
This causes false positive alerts when unschedulable node is no longer accessible, but isn't removed from k8s yet.

## What is the expected result?

`monitoring-ping` no longer tries to ping unschedulable nodes.

## Checklist
- [x] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: monitoring-ping
type: chore
summary: Skip unschedulabe nodes in `monitoring-ping`
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/aws'...

File: modules/340-monitoring-ping/hooks/discover_targets.go [hunk 1]
```diff
@@ -88,8 +92,10 @@ func discoverNodes(input *go_hook.HookInput) error {
 	combinedTargets := newTargets()
 
 	for _, address := range input.Snapshots["addresses"] {
-		convertedAddress := address.(nodeTarget)
-		combinedTargets.Cluster = append(combinedTargets.Cluster, convertedAddress)
+		if address != nil {
+			convertedAddress := address.(nodeTarget)
+			combinedTargets.Cluster = append(combinedTargets.Cluster, convertedAddress)
+		}
```
Comment (RomanenkoDenys) [inline modules/340-monitoring-ping/hooks/discover_targets.go:98]:
```suggestion
        if address == nil {
          continue
        }  
	
	   convertedAddress := address.(nodeTarget)
	   combinedTargets.Cluster = append(combinedTargets.Cluster, convertedAddress)
```

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
small suggestion

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/static'...

**Review by RomanenkoDenys — APPROVED** [review]

**Pull request merged**